proto/buf.yaml

```diff
@@ -1,9 +1,11 @@
-# Generated by "buf config migrate-v1beta1". Edit as necessary, and
-# remove this comment when you're finished.
-#
-# This module represents the "proto" root found in
-# the previous configuration.
 version: v1
+name: buf.build/cosmos/ibc-go
+deps:
+  # see: (https://github.com/cosmos/cosmos-sdk/tree/main/proto#sdk-x-buf)
+  - buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176
```

Comment:
This will need to be updated for v0.47. As per comment above, see: https://github.com/cosmos/cosmos-sdk/tree/main/proto#sdk-x-buf